app/views/propertyrepresentation/appoint/appointAgentProperties.scala.html

```diff
@@ -194,7 +194,7 @@ <h2 class="heading-secondary margin-top-30">@messages("dashboard.managePropertie
                     <ul class="pagination" style="margin: 0px">
                         @if(pagination.page > 1) {
                             <li class="previous ">
-                                <a href="@agentAppointment.routes.AppointAgentController.getMyOrganisationPropertyLinksWithAgentFiltering(pagination.previousPage, params, agentCode, agentAppointed, backLinkAsString).url">
+                                <a href="@agentAppointment.routes.AppointAgentController.getMyOrganisationPropertyLinksWithAgentFiltering(pagination.previousPage, params, agentCode, agentAppointed, backLinkAsString).url "role="link">
```

Comment:
please remove the extra quote

Reply:
i can see now its not an extra quote, it just looks like that because there is a space after the. url and no space after the closing quote